src/server/index.ts

```diff
@@ -117,14 +156,19 @@ initDb()
     // Log http requests
     app.use(morgan(MORGAN_LOG_FORMAT))
 
+    const redirectController = container.get<RedirectControllerInterface>(
+      DependencyIds.redirectController,
+    )
     // API configuration
     app.use('/api', ...apiSpecificMiddleware, api) // Attach all API endpoints
+    app.get(
+      '/assets/transition-page/js/gtag.js',
```

Comment:
would it be cleaner to serve the file from a static endpoint instead?

Reply:
Not possible given that we do not want to have our GA tracking ids as part of the contents of the file. _Maybe_ we could think about spitting out a static file from GA as part of the build... but that is too much work for a very specific part of this codebase